controllers/auth.controller.js

```diff
@@ -19,7 +19,8 @@ const login = async (req, res, next) => {
     try {
         const { email, password } = req.body;
         const data = await userService.login(email, password);
-        res.cookie("refresh_token", data.refreshToken, { maxAge: 24 * 60 * 60 * 1000, httpOnly: true });
+        res.cookie("refresh_token", data.refreshToken, { maxAge: 24 * 60 * 60 * 1000, httpOnly: true, sameSite: "lax" });
```

Comment:
@ikadushkina You use this string twice. To to observe DRY principle, you can move it to a helper or service and use where you need, something like 
`refreshTokenService.set(res, data.refreshToken);`
and
`refreshTokenService.clear(res);`